scripts/install.js

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env node
 
-var npmArgs = JSON.parse(process.env.npm_config_argv);
+var npmArgs = JSON.parse(process.env.npm_config_argv).original;
 
-if (npmArgs.original.indexOf('--bootstrap') === -1) {
+if (npmArgs.indexOf('--create') === -1 && npmArgs.indexOf('--bootstrap') === -1) {
```

Comment:
`--create` is shorter to type and more explicit I find. For now, we can support both, but would suggest we start moving towards `--create` (adjusted the documentation accordingly)